data-model.d.ts

```diff
@@ -73,6 +73,10 @@ export interface Literal {
      * @link http://tools.ietf.org/html/bcp47
      */
     language: string;
+    /**
+     * the direction of the language-tagged string.
+     */
+    direction?: 'ltr' | 'rtl' | '' | null;
```

Comment:
JS noob question: The specification states `direction?: DOMString`. Does it allow `null`?

Reply:
I'm not too sure about the semantics of `direction?: DOMString` in ReSpec.
But at least the text in https://github.com/rdfjs/data-model-spec/pull/175/ says `where <code>direction</code> is optional or can be falsy`.
Falsy includes `null` and `undefined` in JavaScript.

Reply:
Good point! Should we ensure that the WebIDL definition is synchronized with the specification text before merging both MRs?

Reply:
`null` and `undefined` are both implicitly included in other parts of the spec, such as [`Term.equals()`](http://rdf.js.org/data-model-spec/#term-interface).
So this would probably be part of a broader change.
Might be good to open a separate issue for this?

Reply:
Ok! Make sense! Thank you!